Source/WebKit/UIProcess/API/Cocoa/WKWebsiteDataStore.h

```diff
@@ -83,6 +83,30 @@ WK_CLASS_AVAILABLE(macos(10.11), ios(9.0))
 /*! @abstract Returns the cookie store representing HTTP cookies in this website data store. */
 @property (nonatomic, readonly) WKHTTPCookieStore *httpCookieStore WK_API_AVAILABLE(macos(10.13), ios(11.0));
 
+/*! @abstract Get identifier for a data store.
+ @discussion Default and non-persistent data store do not have identifier.
```

Comment:
data store => data stores
have identifier => have an identifier

When we say "do not have", do we mean this property returns nil? If so, let's say nil.